pkg/minikube/translate/translations/fr-FR.json

```diff
@@ -0,0 +1,90 @@
+{
+	"%s: %v": " ",
+	"Advice:        %s": " ",
+	"Alternatively, you may delete the existing VM using `minikube delete -p %s`": " ",
+	"Configuring environment for Kubernetes %s on %s %s": " ",
+	"Configuring local host environment ...": " ",
+	"Creating %s VM (CPUs=%d, Memory=%dMB, Disk=%dMB) ...": " ",
+	"Creating mount %s ...": " ",
+	"Deleting %q from %s ...": " ",
+	"Documentation: %s": " ",
+	"Done! kubectl is now configured to use %q": " ",
+	"Download complete!": " ",
+	"Downloading %s %s": " ",
+	"Downloading Minikube ISO ...": " ",
+	"Error getting machine status": " ",
+	"Error restarting cluster": " ",
+	"Error starting cluster": " ",
+	"Error starting mount": " ",
+	"Error writing mount pid": " ",
+	"Error:         [%s] %v": " ",
+	"Failed to cache ISO": " ",
+	"Failed to cache binaries": " ",
+	"Failed to cache images": " ",
+	"Failed to check if machine exists": " ",
+	"Failed to check main repository and mirrors for images for images": " ",
+	"Failed to chown %s: %v": " ",
+	"Failed to enable container runtime": " ",
+	"Failed to generate config": " ",
+	"Failed to get bootstrapper": " ",
+	"Failed to get command runner": " ",
+	"Failed to get driver URL": " ",
+	"Failed to get machine client": " ",
+	"Failed to save config": " ",
+	"Failed to set NO_PROXY Env. Please use `export NO_PROXY=$NO_PROXY,%s`.": " ",
+	"Failed to setup certs": " ",
+	"Failed to setup kubeconfig": " ",
+	"Failed to update cluster": " ",
+	"For best results, install kubectl: https://kubernetes.io/docs/tasks/tools/install-kubectl/": " ",
+	"For more information, see:": " ",
+	"If the above advice does not help, please let us know: ": " ",
+	"Ignoring --vm-driver=%s, as the existing %q VM was created using the %s driver.": " ",
+	"Kubernetes downgrade is not supported, will continue to use %v": " ",
+	"Launching Kubernetes ... ": " ",
+	"None of known repositories in your location is accessible. Use %s as fallback.": " ",
+	"None of known repositories is accessible. Consider specifying an alternative image repository with --image-repository flag": " ",
+	"Please enter a value:": " ",
+	"Powering off %q via SSH ...": " ",
+	"Pulling images ...": " ",
+	"Re-using the currently running %s VM for %q ...": " ",
+	"Related issues:": " ",
+	"Relaunching Kubernetes %s using %s ... ": " ",
+	"Requested disk size (%dMB) is less than minimum of %dMB": " ",
+	"Restarting existing %s VM for %q ...": " ",
+	"Sorry that minikube crashed. If this was unexpected, we would love to hear from you:": " ",
+	"Sorry, the --gpu feature is currently only supported with --vm-driver=kvm2": " ",
+	"Sorry, the --hidden feature is currently only supported with --vm-driver=kvm2": " ",
+	"Stopping %q in %s ...": " ",
```

Comment:
I now see why you were worried about context. I wonder if it's possible for us to provide a way so that callers can give translators more of a hint, such as:

```Stopping %(name of service)s... in %(name of host)q ...```

Reply:
This is necessary and will be added in a future PR.